angular/modules/unit-test/README-ENG.md

```diff
@@ -15,37 +16,15 @@ Practice - 4 hours
 ## Theory
 
 - [Lecture](https://youtu.be/DCw-JBll2u0?t=2201)
-- [Examples from video](https://github.com/pavelrazuvalau/angular-lectures/tree/master/angular-unit-testing)
-- [TODO List app](https://github.com/pavelrazuvalau/todo-list-management)
-- [Slides](https://slides.com/pavelrazuvalau/angular-unit-testing)
-- [Testing service (RU)](https://medium.com/fafnur/%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5-%D1%81%D0%B5%D1%80%D0%B2%D0%B8%D1%81%D0%BE%D0%B2-%D0%B2-angular-%D1%81-%D0%BF%D0%BE%D0%BC%D0%BE%D1%89%D1%8C%D1%8E-jest-%D1%82%D0%B5%D1%81%D1%82%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D0%BD%D0%B8%D0%B5-%D1%80%D0%B5%D0%B0%D0%BA%D1%82%D0%B8%D0%B2%D0%BD%D0%BE%D0%B9-%D0%B0%D1%81%D0%B8%D0%BD%D1%85%D1%80%D0%BE%D0%BD%D0%BD%D0%BE%D0%B9-%D0%BB%D0%BE%D0%B3%D0%B8%D0%BA%D0%B8-396ba5eca147)
-- [Testings NgRx store](https://itnext.io/complete-testing-of-angular-ngrx-store-with-jest-a4ac5fb55e23)
-- [NgRx testing](https://next.ngrx.io/guide/store/testing)
-- [Testing standalone component (v. 16)](http://www.kamilkonopka.eu/posts/testing-angular-16-standalone-components-with-jest)
-- [HTTP based service (v. 16)](https://blog.stackademic.com/testing-http-based-services-within-angular-16-with-jest-c9e867e22632)
-- [Component testing scenarios](https://angular.dev/guide/testing/components-scenarios)
-- [Testing services](https://angular.dev/guide/testing/services)
+- [Examples from the lecture](https://github.com/pavelrazuvalau/angular-lectures/tree/master/angular-unit-testing)
+- [TODO List Management App (includes tests)](https://github.com/pavelrazuvalau/todo-list-management)
+- [Slides: Unit Testing in Angular](https://slides.com/pavelrazuvalau/angular-unit-testing)
 
 ## Practical assignment
 
-- [Angular. Unit testing with Jest](../../../tasks/angular/unit-testing-jest.md)
-- Test: Angular test (Unit test)
+- TBD
+- Test: Angular test (Unit Testing)
 
 ## Additional materials
 
-- [Testing](https://angular.dev/guide/testing)
```

Comment:
Let's keep some: 
- [Testing](https://angular.dev/guide/testing)
- [Integrate Jest into an angular application](https://timdeschryver.dev/blog/integrate-jest-into-an-angular-application-and-library)
- [Testing NgRx](https://ngrx.io/guide/store/testing)